### PR TITLE
Updated SParkle control.

### PR DESCRIPTION
**Pull request by sp9bsl**

### CODE REVIEW

**Comment by db4ple** [comment]
Will merge and test tomorrow, looks so far good to me.

**Comment by sp9bsl** [comment]
Thanks, I hope not destroyed anything with Andreas board. There will be some refactoring needed around RFboard.CodecRestart I think.
Anyway, no hurry - take your time :)

File: mchf-eclipse/hardware/uhsdr_board.c [hunk 1]
```diff
@@ -247,7 +247,7 @@ void Board_InitMinimal()
     // Minimally initialize the RF board
 
     // Initialize LO, by which we (at least for now) can detect the RF board
-    Osc_Init();
+    //Osc_Init();
```
Comment (db4ple) [inline mchf-eclipse/hardware/uhsdr_board.c:250]: We remove this entirely later.

File: mchf-eclipse/drivers/ui/radio_management.c [hunk 2]
```diff
@@ -1116,6 +1116,15 @@ const int BAND_FILTER_NUM = sizeof(mchf_rf_bandFilters)/sizeof(BandFilterDescrip
  */
 static void RadioManagement_SetHWFiltersForFrequency(uint32_t freq)
 {
+#ifdef USE_OSC_SParkle
```
Comment (db4ple) [inline mchf-eclipse/drivers/ui/radio_management.c:1119]: This is a candidate for another "dynamic" function of  RFBoard interface.

**Review by db4ple — APPROVED** [review]
Comments are for me

**Pull request merged**